client/app/error/error.routes.js

```diff
@@ -0,0 +1,19 @@
+import template from './error.html';
+import controller from './error.controller';
+
+angular.module('managerApp').config(($stateProvider) => {
+  // account module
+  $stateProvider.state('error', {
+    url: '/error',
+    params: {
+      error: null,
+    },
+    template,
+    controller,
+    controllerAs: '$ctrl',
```

Comment:
Can you use a `Component` instead ?

Reply:
What do you mean by ```Component```?

Reply:
Like this !

https://github.com/ovh-ux/manager/blob/3e41b2ce3e87ab0ae2acc9c5d319ce72b9607db0/packages/manager/modules/telecom-task/src/task.component.js